src/amperity/gocd/secret/vault/logging.clj

```diff
@@ -17,7 +17,7 @@
 (defmacro ^:private deflevel
   "Define logging functions for the given level."
   [level]
-  ^:cljfmt/ignore
+  ^:cljstyle/ignore
```

Comment:
This might not be necessary anymore, `cljstyle` now ignores function linebreak rules inside syntax quotes.